docs/tracking/google-tag-manager.md

```diff
@@ -1,17 +1,22 @@
 ---
-title: Google Tag Manager (GTM)
-description: Learn how to integrate Featurevisor SDK with Google Tag Manager
+title: Data Capturing via Google Tag Manager (GTM)
+description: Learn how to integrate Featurevisor SDK with Google Tag Manager and enable your experimentation reporting through custom dataLayer events
 ---
+## Intro:
 
-You can send Featurevisor activation events coming from SDK to Google Tag Manager. {% .lead %}
+Being able to report on the outcome of an experimnt by leveraging high-quality data is a key aspect of any business or individual who wants to take a step closer to achieving their goals and offering users an outstanding experience on their platform. This article guides you on how to run experimentation analysis by leveraging your existing frontend dataset and enabling segmentation for bahvioural analysis through an established data collection method: Google Tag Manager. 
 
-## Custom events in GTM
+## Guide:
 
-Before tracking, make sure you have custom events set up in your container.
+## Send an experiment activation event to the dataLayer and collect data via GTM
```

Comment:
shorter sub-headings will be convenient, since the website also shows the table of content as a sidebar on the right side of the page.

may be we can shorten it to `Push activation event to GTM` (or dataLayer).

Reply:
I actually think this needs the context as it's written, otherwise it would miss a part. Idea is: send to DL, harvest with GTM